src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        final int arrayLenght = 6;
+        final FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figureArray = new Figure[arrayLenght];
```

Comment:
we usually avoid having a data type in the name, java is a strongly typed language anyway
```suggestion
        Figure[] figures = new Figure[arrayLenght];
```